admin_guide/ipsec.adoc

```diff
@@ -0,0 +1,174 @@
+[[admin-guide-ipsec]]
+= ipsec
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+IPsec is a mechanism for encrypting communication between hosts which can
+already communicate with each other via the Internet Protocol (IP). The simplest
+way of protecting traffic within an Openshift cluster is to ensure that the
+master and all nodes use IPsec to encrypt their communication.  This document
+describes the simplest way to do this.
+
+In this example, we will secure communication of an entire IP subnet from which
+the Openshift hosts receive their IP addresses. Since you are securing the
+host-to-host communication, this automatically includes all cluster management
+and pod data traffic.
+
+[[requirements]]
+== Requirements
+This guide requires that you have the `libreswan` package, version 3.19 or later,
+installed on cluster hosts.  Only 3.19 and later include the necessary
+opportunistic group functionality that allows hosts to be configured without
+knowledge of every other host in the cluster.
+
+[[procedure]]
+== Procedure
+This procedure should be repeated for each host in your cluster.  Any host
+in the cluster that does not have IPsec enabled will not be able to communicate
+with a host that does. It is suggested to perform these steps on the master
+first, and then each node.
+
+[[certificates]]
+=== Certificates
+By default, Openshift secures cluster management communication with mutually
+authenticated HTTPS communication. This means that both the client (like an
+openshift node) and the server (like the openshift api-server) send each other
+their certificates, which are checked against a know Certificate Authority (CA).
+These certificates are generated at cluster setup time and typically live on
+each host.
+
+These certificates can also be used to secure pod communications with IPsec. You
+need at three files on each host:
```

Comment:
either an extra or a missing word there (depending on whether you meant "at least three" or "three")